src/main/kotlin/org/rust/cargo/runconfig/test/CargoTestEventsConverter.kt

```diff
@@ -242,8 +242,8 @@ class CargoTestEventsConverter(
                 .addAttribute("details", failedMessage)
             val (left, right) = extractDiffResult(failedMessage) ?: return builder
             return builder
-                .addAttribute("expected", left)
-                .addAttribute("actual", right)
+                .addAttribute("left", left)
+                .addAttribute("right", right)
```

Comment:
These constants are not related to UI labels. They are names of concrete fields of a test event
So we should use `expected` and `actual` here